buildroot-external/board/hardkernel/odroid-n2/patches/uboot/0001-HACK-mmc-meson-gx-limit-f_max-to-24-MHz-on-the-first.patch

```diff
@@ -1,4 +1,4 @@
-From 024796cbf752d2e210341ae8609792803641eb92 Mon Sep 17 00:00:00 2001
+From f63ef87b9a9bf9997aa4017f8380567b20a1810c Mon Sep 17 00:00:00 2001
 From: =?UTF-8?q?Jan=20=C4=8Cerm=C3=A1k?= <sairon@sairon.cz>
 Date: Thu, 7 Nov 2024 12:39:02 +0100
 Subject: [PATCH] HACK: mmc: meson-gx: limit f_max to 24 MHz on the first try
```

Comment:
From looking at https://github.com/home-assistant/operating-system/commit/e42835f1634fbedf504bff92c3282e98c4db20ca, this might also be needed for ODROID-C4 :thinking: